SU2_CFD/src/drivers/CDiscAdjMultizoneDriver.cpp

```diff
@@ -665,6 +665,7 @@ void CDiscAdjMultizoneDriver::SetRecording(RECORDING kind_recording, Kind_Tape t
                                                          config_container, iZone, INST_0);
       AD::Push_TapePosition(); /// leave_zone
     }
+    Print_DirectResidual(kind_recording);
```

Comment:
I made that bit a function for Multizone just as there is for SingleZone. Using the same function for both is maybe possible without to much effort. But I did not do that for now

Reply:
Did that, see latest commit https://github.com/su2code/SU2/pull/1371/commits/af7d12c2e6621f793098f201885db248e72da533